### PR TITLE
Build gtk from freedesktop sdk with custom patches

### PR DESCRIPTION
**Pull request by xjuan**
Add gtk3-fix-atk-gjs-crash.patch to fix bug #T19642

https://phabricator.endlessm.com/T19642

### CODE REVIEW

**Comment by nedrichards** [comment]
fwiw you tagged the wrong ticket ;-)

**Comment by xjuan** [comment]
yes, thanks

File: com.endlessm.apps.Sdk.json.in [hunk 1]
```diff
@@ -41,6 +45,18 @@
 
     "modules": [
         {
+            "name": "clean-out-old-gstreamer",
```
Comment (ebassi) [inline com.endlessm.apps.Sdk.json.in]: This is not needed any more: we use the appropriate build from the SDK.

File: clean-out-old-gstreamer-makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+PLUGINS_TO_REMOVE = $(addprefix /usr/lib/gstreamer-1.0/libgst,$(addsuffix .so,\
```
Comment (ebassi) [inline clean-out-old-gstreamer-makefile]: This script is not needed in master/sdk-2

**Review by ebassi — CHANGES_REQUESTED** [review]

**Comment by ebassi** [comment]
Need to update the Docker image we use for CI

**Pull request merged**